packages/lit-virtualizer/src/test/helpers.ts

```diff
@@ -21,19 +21,21 @@ export async function wait(n=0) {
  * Before/After parameters are given to the function to ensure there are no user-error
  * cases where teardown is forgotten/skipped.
  */
-export function ignoreBenignErrors(before, after) {
+export function ignoreBenignErrors(before: Mocha.HookFunction, after: Mocha.HookFunction) {
     ignoreWindowErrors(before, after, /^(ResizeObserver loop limit exceeded)$/);
 }
-export function ignoreWindowErrors(before, after, regexp) {
-    let onerror;
+export function ignoreWindowErrors(before: Mocha.HookFunction, after: Mocha.HookFunction, regexp: RegExp) {
+    let onerror: OnErrorEventHandler;
     before(() => {
         onerror = window.onerror;
         window.onerror = (err) => {
             if (regexp.test(`${err}`)) {
                 console.warn(`Ignored Error: ${err}`);
                 return false;
             }
-            return onerror.apply(window, arguments);
+            if (onerror) {
+                return onerror.apply(window, [...(arguments as unknown as [event: string | Event, source?: string | undefined, lineno?: number | undefined, colno?: number | undefined, error?: Error | undefined])]);
```

Comment:
Instead of listing out the arguments, this can be replaced with:

`arguments as unknown as Parameters<typeof onerror>`.

I.e. TypeScript playground: https://www.typescriptlang.org/play?#code/FAMwrgdgxgLglgewgAgE4ENXIBQEpkDewyJyANgKYzJIWqoKoAyCU6ZyAvMgO5wQATBDwB0teowCEAbmKlxDZq3Yj0ABzVkAntj6DhAGmSYA5mAC2FCDADOxu5ADWEYSnR2ACpnSWYdGwA8MFpqFAggNBB0iixsZAB8uLIAvsBAA

Reply:
That is better.  Thanks!